client/landing/stepper/declarative-flow/internals/steps-repository/store-profiler/index.tsx

```diff
@@ -41,49 +45,44 @@ const StoreProfiler: Step = function StoreProfiler( { navigation } ) {
 		) );
 	}, [ verticals ] );
 	// TODO: This may not be the right source for this info.
```

Comment:
Does this TODO still apply after switching to the `useCountriesAndStates()` hook?

Reply:
I don't think so - removing!